### PR TITLE
tod: add page

### PR DESCRIPTION
**Pull request by Coriakin**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/tod.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# tod
+
+> A tiny Todoist command line client in Rust.
```
Comment (kbdharun) [inline pages/linux/tod.md:3]:
```suggestion
> A tiny Todoist client in Rust.
```

File: pages/linux/tod.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+> Takes simple input and dumps it in your inbox or anoclearther project. Takes advantage of natural language processing to assign due dates, tags, etc.
+> More information: <https://github.com/alanvardy/tod>.
+
+- Import your projects, this is necessary to enable project prompts:
```
Comment (kbdharun) [inline pages/linux/tod.md:7]:
```suggestion
- Import your projects (this is necessary to enable project prompts):
```

File: pages/linux/tod.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+`tod task next`
+
+- Get your work schedule:
+
+`tod task list --scheduled --project work`
+
+- Get all tasks for work:
+
+`tod task list --project work`
```
Comment (kbdharun) [inline pages/linux/tod.md:33]:
```suggestion
`tod task list --scheduled --project {{work}}`

- Get all tasks for work:

`tod task list --project {{work}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by kbdharun — APPROVED** [review]

**Review by blueskyson — APPROVED** [review]

**Pull request merged**